### PR TITLE
Please Ignore: Testing the GitHub App access

### PR DESCRIPTION
**Pull request by oleg-nenashev**

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Now it should use the app

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Plugin Installation Manager Tool
+Plugin Installation Manager Tool (just4test)
```
Comment (oleg-nenashev) [inline README.md:1]:
```suggestion
Plugin Installation Manager Tool (just4test2)
```

**Comment by jetersen** [comment]
🤔 I cannot ignore this, what are we testing? Sounds interesting 😄

**Comment by timja** [comment]
> 🤔 I cannot ignore this, what are we testing? Sounds interesting 😄

Using github app for auth so that we can enable this soon: 

https://github.com/jenkinsci/github-checks-plugin

(along with higher rate limits)

**Comment by timja** [comment]
The builds are no longer triggered by jenkins admin they are triggered by https://github.com/apps/ci-jenkins-io

**Comment by jetersen** [comment]
Great work!

**Pull request closed without merging**